updated changelog.md for release 1.4.0 (#2901)
redhat-developer/vscode-openshift-tools: CHANGELOG.md

# Change Log
## 1.4.0 (May 23, 2023)

> * [#2798](https://github.com/redhat-developer/vscode-openshift-tools/issues/2798) Generation of tools.json file with updated cli tool version and sha256 checksums for new release

Reviewer: ```suggestion
* [#2798](https://github.com/redhat-developer/vscode-openshift-tools/issues/2798) Auto generation of tools.json file with updated cli version and sha256 checksums
```

* [#709](https://github.com/redhat-developer/vscode-openshift-tools/issues/709) Login to Cluster command should check if cluster is started
* [#2861](https://github.com/redhat-developer/vscode-openshift-tools/pull/2861) Add 'Sign in with Red Hat' item to Accounts menu when extension activated
* [#2864](https://github.com/redhat-developer/vscode-openshift-tools/issues/2864) 'OpenShift: Create' fails with "You need at least one project available"

> * [#2870](https://github.com/redhat-developer/vscode-openshift-tools/pull/2870) Bump ODO 3.10.0

Reviewer: ```suggestion
* [#2870](https://github.com/redhat-developer/vscode-openshift-tools/pull/2870) Support odo 3.10.0 workflow
```

> * [#2869](https://github.com/redhat-developer/vscode-openshift-tools/issues/2869) Improved the Helm Chart UX

Reviewer: ```suggestion
* [#2869](https://github.com/redhat-developer/vscode-openshift-tools/issues/2869) New Improved Helm Chart workflow
```
